### PR TITLE
Combine references

### PR DESCRIPTION
**Pull request by thomasbtf**
This PR introduces improved filtering of reads, based on combined reference from Human and Wuhan genomes.

### CODE REVIEW

**Comment by thomasbtf** [comment]
After assembly with human reference: 

`26 contigs, total 29010 bp, min 308 bp, max 3552 bp, avg 1115 bp, N50 1323 bp`

After assembly with main and human reference: 

`91 contigs, total 19973 bp, min 206 bp, max 221 bp, avg 219 bp, N50 220 bp`

**Comment by thomasbtf** [comment]
With the 91 contigs, raGOO is not able to create a pseudo chromosome.

**Comment by thomasbtf** [comment]
I think the problems lies within the filtering with Samtools in the rule extract_nonhuman_reads. Before we apply this step, 1.6 million of the reads are identified as SARS-related coronaviruses. After filtering this number reduces to 1 read.

File: workflow/rules/qc.smk [hunk 1]
```diff
@@ -133,7 +148,8 @@ rule extract_nonhuman_reads:
         (samtools view  -@ {threads} -u -f 4 -F 264 {input} > {output.t1}) 2> {log}
         (samtools view  -@ {threads} -u -f 8 -F 260 {input} > {output.t2}) 2>> {log}
         (samtools view  -@ {threads} -u -f 12 -F 256 {input} > {output.t3}) 2>> {log}
-        samtools merge -@ {threads} -n {output.unsorted} {output.t1} {output.t2} {output.t3} >> {log} 2>&1
+        (samtools view  -@ {threads} -h -f 1 -F 12 {input} | grep 'NC_045512.2' | samtools view -u > {output.t4}) 2> {log}
```
Comment (thomasbtf) [inline workflow/rules/qc.smk:151]: It seems when filtering for the covid reads with Samtools view, the 

> `-f 2` (PROPER_PAIR: each segment properly aligned according to the aligner)

 is too strict on the test reads.

When using the flag 

> `-f 1` (PAIRED: paired-end (or multiple-segment) sequencing technology) 

more covid reads of the test data are accpeted.

**Review by alethomas — APPROVED** [review]

**Pull request merged**